modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -542,6 +540,8 @@ private void setAspnetCoreVersion(String packageFolder) {
         } else if ("2.0".equals(aspnetCoreVersion.getOptValue())) {
             supportingFiles.add(new SupportingFile("web.config", packageFolder, "web.config"));
             compatibilityVersion = null;
+        } else if ("2.2".equals(aspnetCoreVersion.getOptValue()) || "2.1".equals(aspnetCoreVersion.getOptValue())) {
+            supportingFiles.add(new SupportingFile("wwwroot" + File.separator + "web.config", packageFolder + File.separator + "wwwroot", "web.config"));
```

Comment:
The web.config under wwwroot is no longer required for ASP.NET Core 2.1 / 2.2.

Reply:
We can remove it in the upcoming major release 5.0.x.

The upcoming release 4.3.1 is a patch release (without breaking changes)

Reply:
@shibayan PM me on Slack if you've further question on this.